test/format/format.h

```diff
@@ -357,8 +357,9 @@ typedef struct {
 
     SAP sap; /* Thread's session event handler information */
 
-    WT_RAND_STATE data_rnd;  /* thread RNG state for data operations */
-    WT_RAND_STATE extra_rnd; /* thread RNG state for extra operations */
+    WT_RAND_STATE data_rnd;    /* thread RNG state for data operations */
+    WT_RAND_STATE extra_rnd;   /* thread RNG state for extra operations */
+    WT_RAND_STATE thread_rnd;  /* thread RNG state independent of format.independent_thread_rng */
```

Comment:
I think we should use `extra_rnd` rather than having a new rand.   Is there a reason not to?  `data_rnd` is used only for changes to data, that allows us to rerun and have the same stuff on disk every time.  `extra_rnd` is for everything else.

Reply:
These ones unfortunately depend on `format.independent_thread_rng` (see comment above).

Reply:
I'm wondering if tying it to `format.independent_thread_rng` might be a feature?  If you are approaching this work as benchmarking, then you definitely don't want your throttles coordinated between, better to have it spread out.  But the point of test/format is randomized testing, not benchmarking.  And in that world, isn't it better to have some runs randomly decide to have coordinated RNGs (and thus start out with throttling between threads somewhat coordinated)?  See this comment from ops.c:
```
    /*
     * Characterize the per-thread random number generator. Normally we want independent behavior so
     * threads start in different parts of the RNG space, but we've found bugs by having the threads
     * pound on the same key/value pairs, that is, by making them traverse the same RNG space. 75%
     * of the time we run in independent RNG space.
     */
```